Firestore/core/src/firebase/firestore/util/executor_libdispatch.mm

```diff
@@ -289,17 +309,18 @@ void MarkDone() {
 ExecutorLibdispatch::PopFromSchedule() {
   absl::optional<Executor::TaggedOperation> result;
 
-  RunSynchronized(this, [this, &result] {
+  RunSynchronized(this, [this, &result]() -> void {
```

Comment:
Hmm, I'm surprised an explicit return type is needed here, or is it just linter complaining?

Reply:
`clang-tidy` was complaining.